### ISSUE TITLE
Can't update with composer and Bedrock

### ISSUE THREAD

**Issue by jwittorf** [afragen/git-updater#960]
Hi,

I'm using https://roots.io/bedrock/ as WordPress boilerplate with composer and it worked fine with the GitHUB updater.

However, if I remove version 9 and add version 10 (Git Updater) I'm getting this on every page:

```
Fatal error: Uncaught Error: Call to undefined function get_theme_root() in /mnt/l/htdocs/wp.demo/web/app/plugins/git-updater/vendor/freemius/wordpress-sdk/start.php:56

Stack trace:
#0 /mnt/l/htdocs/wp.demo/vendor/composer/autoload_real.php(71): require() 
#1 /mnt/l/htdocs/wp.demo/vendor/composer/autoload_real.php(61): composerRequire0ee8b5d549268db9fd171817e4e05b42('a80dab40a30760c...', '/mnt/l/htdocs/w...') 
#2 /mnt/l/htdocs/wp.demo/vendor/autoload.php(7): ComposerAutoloaderInit0ee8b5d549268db9fd171817e4e05b42::getLoader() 
#3 /mnt/l/htdocs/wp.demo/web/wp-config.php(7): require_once('/mnt/l/htdocs/w...') 
#4 /mnt/l/htdocs/wp.demo/web/wp/wp-load.php(42): require_once('/mnt/l/htdocs/w...') 
#5 /mnt/l/htdocs/wp.demo/web/wp/wp-admin/admin.php(34): require_once('/mnt/l/htdocs/w...') 
#6 /mnt/l/htdocs/wp.demo/web/wp/wp-admin/update-core.php(10): require_once('/mnt/l/htdocs/w...') 
#7 {main} thrown in /mnt/l/htdocs/wp.demo/web/app/plugins/git-updater/vendor/freemius/wordpress-sdk/start.php on line 56
```

When I open the file, my PhpStorm IDE find's the function and actually lets me jump to the theme.php, I've already ran `composer dump-autoload` but that didn't change anything.

Do you have any tips or hints to solve it?

Thanks in advance!

**Comment by afragen** [comment]
It seems that Bedrock isn't loading `wp-includes/theme.php` early enough in the loop to have this function defined. This file is normally included very early in the `wp-settings.php` file. This is where the function `get_theme_root()` is defined.

Unfortunately I am not familiar with Bedrock. You could try adding `require_once ABSPATH . WPINC . '/theme.php';` somewhere early in the Bedrock loading of WordPress.

**Comment by afragen** [comment]
I found this. https://discourse.roots.io/t/bedrock-composer-autoloader-and-wordpress/4343

Possibly adding a mu-plugin that loads the necessary files might work.

**Comment by afragen** [comment]
I think I may have a solution. Which of the Git Updater family of plugins are you currently using?

**Comment by afragen** [comment]
Can you try using the `develop` branch and see if the error has been fixed? Thanks.

**Comment by afragen** [comment]
Now released in v10.3.3

**Comment by jwittorf** [comment]
Wow, thanks for your effort! I'll try it in the next days and will keep you posted.

P.S.: I was simply using the GitHub-Updater, nothing more.
